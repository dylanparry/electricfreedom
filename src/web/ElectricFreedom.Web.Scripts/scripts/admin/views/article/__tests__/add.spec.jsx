import React from 'react';
import { shallow } from 'enzyme';

import ArticleAdd from '../add';

const setup = () =>
{
  const wrapper = shallow(<ArticleAdd />);

  return {
    wrapper,
  };
};

describe('ArticleAdd', () =>
{
  test('render', () =>
  {
    const { wrapper } = setup();

    expect(wrapper.exists()).toBe(true);
  });
});
