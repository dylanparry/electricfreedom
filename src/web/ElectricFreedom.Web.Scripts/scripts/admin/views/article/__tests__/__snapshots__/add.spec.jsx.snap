// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleAdd render 1`] = `
<p>
  Add Article
</p>
`;
